README.md

```diff
@@ -115,6 +115,8 @@ If both are specified, APPSODY_CONTROLLER_IMAGE will be used.
 
 If the appsody stack of interest uses a script file (.sh for example) that is then edited by the `vi` editor while the script is running, the file modification time is not updated on the container file system until the script ends.  What this means is that the ON_CHANGE action is not triggered when `vi` writes the file.
 
+The vender watcher package code used by the controller uses filepath.Walk function from the path/filepath package to traverse the file system.  This function does not evaluate symlinks.  If the APPSODY_WATCH_DIR is a symlink or contains files and directories that are symlinks, it is possible that unexpected behavior may occur during file watching as the watcher may not be able to traverse symlinks or detect changes in the underlying file referenced by the symlink as expected.
+
```

Comment:
Perhaps each known issue should have a small title and then other details can be text under the title?

Reply:
The first sentence is a little impenetrable to the reader. I'd suggest simpler text such as:

"The use of symlinks within the directories that the controller monitors for file changes might result in undefined behavior."

If you think that additional explanation is needed try something more like the following:

"The controller code that defines how to traverse the file system does not evaluate symlinks. Therefore, if the `APPSODY_WATCH_DIR` is a symlink, or contains files and directories that are symlinks, unexpected behavior might occur when the controller monitors for file changes."